__tests__/app.test.js

```diff
@@ -239,4 +245,59 @@ describe("POST /api/articles/:article_id/comments", () => {
     expect(updated.rows.length).toBe(original.rows.length + 1);
     expect(updated.rows.some((obj) => obj.body === "hello5")).toBe(true);
   });
-});
\ No newline at end of file
+});
+describe("PATCH /api/articles/:article_id", () => {
+  test("returns a 400 error for incorrect id type", async () => {
+    const post = { inc_votes: 6 };
+    const response = await request(app)
+      .patch("/api/articles/banan")
+      .send(post)
+      .expect(400);
+    expect(response.body.msg).toBe("400 Bad Request");
+  });
+  test("returns a 404 error for id that does not exist", async () => {
+    const post = { inc_votes: 6 };
+    const response = await request(app)
+      .patch("/api/articles/85")
+      .send(post)
+      .expect(404);
+    expect(response.body.msg).toBe("404 article_id of 85 Not Found");
+  });
+  test("returns 400 error for invalid request body", async () => {
+    const post = {};
+    const response = await request(app)
+      .patch("/api/articles/5")
+      .send(post)
+      .expect(400);
+    expect(response.body.msg).toBe("400 Bad Request");
+  });
```

Comment:
Please add another test where the 'inc_votes' property has the wrong data type (not a number). Think about what error code would that be.